BitwardenShared/Core/Platform/Services/ConfigService.swift

```diff
@@ -138,6 +145,17 @@ class DefaultConfigService: ConfigService {
         return localConfig
```

Comment:
⚠️ We need to also call loading flags here because as soon as the app starts if it's not forcing refresh, the `localConfig` is not `nil` and is not expired then it's not going to perform what's inside the `if` thus it won't load the flags which would be wrong. So we always need to update the flags with the latest version of the config, if there's one:
```suggestion
        if let localConfig {
            await loadFlags(localConfig)
        }
        return localConfig
```